frontend/alert/components/AutoComplete.tsx

```diff
@@ -260,6 +213,13 @@ const AutoComplete = ({ defaultValue="", onValueChange, setTimeline, disabled })
         }
     }, [suggestionsFromBackend, value]);
 
+    // Create input text for bulk mode in the form of: [last selected section title] + [# selected courses - 1] more
+    const generateCoursesValue = () => {
+        let lastElement;
+        for (lastElement of Array.from(selectedCourses));
```

Comment:
Could just do `const lastElement = Array.from(selectedCourses).pop();`

Reply:
Doing this gives me `Type 'Section | undefined' is not assignable to type 'Section', Type 'undefined' is not assignable to type 'Section'` (although lastElement cannot be undefined..) which is why I decided to loop over it. Any suggestions on how to fix this error?

Reply:
Hmm then maybe `const lastElement = Array.from(selectedCourses).pop()!;` would work? This is the [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) (works for `undefined` too).

Reply:
ahh didn't know that, thanks!